anyway/models.py

```diff
@@ -167,14 +182,23 @@ def get_id(self):
         return self.id
 
 
-class Role(Base, RoleMixin):
-    __tablename__ = "roles"
-    id = Column(Integer(), primary_key=True)
-    name = Column(String(80), unique=True)
+class Roles2(Base):
+    __tablename__ = "roles2"
+    id = Column(Integer(), primary_key=True, autoincrement=True, nullable=False)
+    name = Column(String(127), unique=True, index=True, nullable=False)
     description = Column(String(255))
+    create_date = Column(DateTime(), nullable=False)
 
-    def __unicode__(self):
-        return self.name
+
+# This table is not in use - it was create a long time ago and is replaced by 'Roles2'
+# class Role(Base, RoleMixin):
+#     __tablename__ = "roles"
+#     id = Column(Integer(), primary_key=True)
+#     name = Column(String(80), unique=True)
+#     description = Column(String(255))
+#
+#     def __unicode__(self):
+#         return self.name
```

Comment:
I prefer to remove unused code. We can always find it by going to previous commits.